.circleci/config.yml

```diff
@@ -122,6 +129,58 @@ jobs:
           command: cargo clippy -- -D warnings -A clippy::unused-unit -A clippy::from-over-into
       - *save_cache
 
+  deploy_node:
+    docker:
+      - image: circleci/rust
+    steps:
+      - attach_workspace:
+          at: /home/circleci/project
+      - setup_remote_docker:
+          docker_layer_caching: false
+      - run:
+          name: Docker login
+          command: |
+            docker login --username $DOCKER_LOGIN --password $DOCKER_PASSWORD
+
+      # NOTE: make sure no checkout with
+      #       a .dockerignore file ignoring target
+      - run:
+          name: Build basic Dockerfile
+          command: |
+            # We use the artifact of the previous
+            # build step which should be in 
+            #    `build/release/node`
+            # in the backend dir.
+
+            # this escape \<< is circeci specific
+            cat \<<EOS >Dockerfile
+            FROM ubuntu:focal
+
+            WORKDIR /app
+
+            COPY target/debug/node /app
+            ENTRYPOINT ["/app/node"]
+            EOS
+
+            # comment needs to be here
+
+      - run:
+          name: Setup custom environment variables
+          command: |
+            export BUILD_ID="${CIRCLE_SHA1:0:7}_$(date +'%Y%m%d_%H%M')"
+            export IMAGE="boymaas/nodefcl:$BUILD_ID"
```

Comment:
Can we have a TODO somewhere to make a Docker account for fractal and putting the base image there?

Reply:
Yes we can. Will add it.